### PR TITLE
Add various papers

### PR DESCRIPTION
**Pull request by dimkarakostas**
## Checklist for submitting a pull request to `blockchain-papers`:

- [x] I have carefully **read and comply** with the [Contributing Guidelines](https://github.com/decrypto-org/blockchain-papers/blob/master/CONTRIBUTING.md) of this repo.
- [x] I have **searched** the [PRs](https://github.com/decrypto-org/blockchain-papers/pulls) of this repo and believe that this is not a duplicate.

### CODE REVIEW

**Comment by dimkarakostas** [comment]
@dionyziz any review comments?

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ peer-reviewed and presented in academic conferences.
 - 🎓 [Ouroboros Praos: An adaptively-secure, semi-synchronous proof-of-stake protocol](http://eprint.iacr.org/2017/573.pdf). Bernardo D, Gazi P, Kiayias A, Russell A. Crypto '17.
 - 🎓 [Ouroboros Genesis: Composable Proof-of-Stake Blockchains with Dynamic Availability](https://eprint.iacr.org/2018/378.pdf). Badertscher C, Gazi P, Kiayias A, Russell A, Zikas V. '18.
 - 🎓 [A Proof-of-Stake protocol for consensus on Bitcoin subchains](http://eprint.iacr.org/2017/417.pdf). Bartoletti M, Lande S, & Podda A S. FC '17.
+- 🎓 [Snow White: Provably Secure Proofs of Stake.](https://fc19.ifca.ai/preproceedings/141-preproceedings.pdf). Bentov I, Pass R, Shi E FC '19.
```
Comment (dionyziz) [inline README.md]: Needs a "." before "FC"

File: README.md [hunk 2]
```diff
@@ -246,6 +248,7 @@ Sompolinsky Y, Lewenberg Y, Zohar A. '16.
 - [BigchainDB: A Scalable Blockchain Database](https://www.bigchaindb.com/whitepaper/bigchaindb-whitepaper.pdf). McConaghy T, Marques R, Müller A, De Jonghe D, McConaghy T, McMullen G, Henderson R, Bellemare S, Granzotto A. '17
 - 🎓 [Commitcoin: Carbon dating commitments with bitcoin](https://eprint.iacr.org/2011/677.pdf). Clark J, Essex A. FC '12.
 - [OpenTimestamps: Securing Software Updates using the Bitcoin Blockchain Financial Cryptography and Data Security](https://github.com/opentimestamps/). Todd P, Halpin H. (FC Poster '17)
+- 🎓 [Decentralized trusted timestamping using the crypto currency bitcoin](https://arxiv.org/pdf/1502.04015.pdf). Gipp B., Meuschke N., Gernandt A. iConference 2015
```
Comment (dionyziz) [inline README.md]: This doesn't seem very novel. Commitcoin did it a few years before them. Likewise, Peter Todd's OpenTimestamps was developed in 2012, and they don't even seem to mention it in their paper, which is essentially the same idea... I'm not sure we want to include this.

Comment (dimkarakostas) [reply README.md]: Well I don't think we have only very novel works here. This was also published in a venue, so somebody interested in timestamping may wanna take a look for any extra stuff. If you don't have any objections directly on the paper itself (eg mistakes) I think we should add it.

**Comment by dionyziz** [comment]
Ok

**Pull request merged**